<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelAdjustWidgetClass</class>
 <widget class="QWidget" name="LevelAdjustWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LevelAdjustWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="connectAll">
          <property name="text">
           <string>connectAll</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeAll">
          <property name="text">
           <string>closeAll</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="title">
        <string>status</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>connectStatus:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="connectStatus">
            <property name="text">
             <string>Unconnected</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>adjustStatus:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="adjustStatus">
            <property name="text">
             <string>Unadjusted</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="title">
      <string>Current angle</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Collimator(X, Y),unit(s)</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="CollimatorX">
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="CollimatorY">
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>Gradienter(X, Y),unit(Â°)</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="GradienterX">
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="GradienterY">
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Motion3D</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>X,Y,Z(mm)</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="motion3DX">
           <property name="text">
            <string>0.000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>,</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="motion3DY">
           <property name="text">
            <string>0.000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>,</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="motion3DZ">
           <property name="text">
            <string>0.000</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Manual control</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="TipTiltManual">
        <property name="text">
         <string>Manual_TipTilt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="motion3DManual">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Manual_Motion3D</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="AutoParallel">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Automatic horizontal adjustment</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="stop">
     <property name="text">
      <string>Stop Automatic horizontal adjustment</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
